activemodel/lib/active_model/dirty.rb

```diff
@@ -189,7 +189,7 @@ def attribute_was(attr) # :nodoc:
     end
 
     # Handles <tt>*_previously_changed?</tt> for +method_missing+.
-    def attribute_previously_changed?(attr, options = {}) #:nodoc:
+    def attribute_previously_changed?(attr) #:nodoc:
```

Comment:
Looks like the [arg was added for parity with the `attribute_changed?` method signature](https://github.com/rails/rails/pull/19847/files#diff-aaddd42c7feb0834b1b5c66af69814d3R185), despite being unused. It's only been present through 5.0.0 betas. Think we're good to drop it. If the method takes args later, they'll be keyword args anyway.